management-api-server/src/main/java/com/datastax/mgmtapi/resources/v1/NodeOpsResources.java

```diff
@@ -120,4 +121,45 @@ public Response schemaVersions() {
           return Response.ok(schemaVersions).build();
         });
   }
+
+  @POST
+  @Path("/repair")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "202",
+      description = "Job ID for successfully scheduled Cassandra repair request",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
```

Comment:
What's the benefit of having:

```json
{ "jobId": 0ea78490-51cd-466a-9674-1c9adb723e50 }
```

vs.

```
0ea78490-51cd-466a-9674-1c9adb723e50
```

As return payload? It would also be very different from all the other v0/v1 APIs that return jobId.

Reply:
So it is clear what you're returning. We have very explicitly said in our REST API guidelines that we always return JSON.

Reply:
This would be very different from other v0 / v1 APIs, which return jobId as the payload itself and not a wrapper around single field. In v2, we could return the Job object, but here it would basically cause incompatibility with other implementations.

Reply:
I'm fine with it being a simple String here. For this phase, adding in any v0/v1 endpoints should be kept to a bare minimum, and when they are necessary, as the case here, it's fine to not return JSON objects for simple Strings, integers, etc. If the returned data is more complex than a simple type, then we should go through the trouble of creating a JSON object in the models package and fit it here to an extent.

For every endpoint we have in this project, we will fix all of this when v2 endpoints are implemented. For now, jsut get what we need to move forward.

Reply:
@burmanm and I discussed last night and we all agreed that we're "OK" with a string here. Actually it makes my skin crawl, but this isn't a v2 so I'm trying not to be OCD :)